scripts/scil_filter_streamlines_by_orientation.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Script to filter streamlines based on their orientation (total length per
+dimension).
```

Comment:
Suggestion:
Script to filter streamlines based on their distance traveled in a specific dimension (x, y, or z).

For example, it helps differentiate a CC streamline versus Cingulum streamline because of its X versus Y  orientation. 
(give an example command line)